OutOfSchool/OutOfSchool.WebApi/Services/RatingService.cs

```diff
@@ -1,7 +1,7 @@
 ﻿using System.Linq.Expressions;
 using AutoMapper;
-using Castle.Core.Internal;
 using Microsoft.Extensions.Localization;
+using Microsoft.IdentityModel.Tokens;
```

Comment:
Is this using required? Because no changes were introduced in the code, just new using added, and it seems to be irrelevant